library/core/src/lib.rs

```diff
@@ -161,7 +161,10 @@
 #![feature(decl_macro)]
 #![feature(doc_cfg)]
 #![feature(doc_notable_trait)]
-#![feature(doc_primitive)]
+// FIXME: to be removed once rustdoc_internals is merged.
```

Comment:
Please remove these FIXMEs. Whoever updates the bootstrap compiler will update all the uses of `cfg(bootstrap)` :)